### DISCUSSION TITLE
Compile for Android

### DISCUSSION THREAD

**Discussion by ghost** [gojue/ecapture#189] [Q&A] [answered]
Could you please advise me how can I compile the project for Android?

**Comment by cfc4n** [comment, marked as answer]
arm64  ubuntu 20.04 or newer. clone it, used `ANDROID=1 make` or `ANDROID=1 make nocore` to compile.

see https://github.com/ehids/ecapture/blob/master/.github/workflows/release.yml#L131 for more detail.

sometimes, you can buy a cloud server  like AWS/aliyun/QCLOUD  etc to used `ARM` CPU ,and build a androie ELF ecapture. 


OR , use androd OS to build it  by  https://blog.seeflower.dev/archives/172/

**Comment by HorseLuke** [comment]
If you only have x86_64 machine, you can install docker, then run an arm64 container to compile it.

Google keywords: "docker arm64 on x86", then you can see this: https://www.stereolabs.com/docs/docker/building-arm-container-on-x86/
